### ISSUE TITLE
Making the CFG file Launch PTXprint in Windows doesn't work as expected

### ISSUE THREAD

**Issue by davidc86** [sillsdev/ptx2pdf#157]
Seems it would be helpful if a user could click on a CFG file in File Explorer and have PTXprint launch with that configuration. I set File Explorer to do just that, but apparently the default way Windows sets this up does not line up with what PTXprint is expecting. So PTXprint opens, but the configuration file is not loaded.
Also, oddly, when I do this, PTXprint adds a **palaso-python** folder in that configuration's folder! If I open the configuration file from within PTXprint, then the **palaso-python** folder is **_not_** generated.

**Comment by davidg-sil** [comment]
Not being a windows person at all these days,  do you get to specify arguments to a program when you associate a program with a file type? I don't remember what the flag ptxprint expects, (-c )  but there is one.to soecify confugyratuion and another to specify project.
 I guess actually you need to specigy both by interpreting a path so I'm calling this an enhancement!

**Comment by davidc86** [comment]
When we first install PTXprint, I see a "readme" flash on the screen, which gives instructions on how to use it with a command line. But after it installs, I can't find that documentation anywhere. If we could reread that information, it should be fairly trivial to create shortcuts that use these commands correctly. 

In the end though, it would be better to make PTXprint able to launch automatically correctly with a given cfg file and path.

**Comment by mhosken** [comment]
Here's the output from ptxprint -h:
```
usage: ptxprint [-h] [-c CONFIG] [-p PARATEXT] [-d DIRECTORY] [-P] [-b BOOKS] [-m MACROS] [-l] [-e] [--timeout TIMEOUT] [-T]
                [-f FONTPATH] [--nofontcache] [-S] [-A ARCHIVE] [--nuseusfmerge] [--debug]
                [pid]

positional arguments:
  pid                   Project id

optional arguments:
  -h, --help            show this help message and exit
  -c CONFIG, --config CONFIG
                        Configuration path to load
  -p PARATEXT, --paratext PARATEXT
                        Path to Paratext Projects directory
  -d DIRECTORY, --directory DIRECTORY
                        Directory to store temporary files in
  -P, --print           Hits print
  -b BOOKS, --books BOOKS
                        Space separated bookid list to set
  -m MACROS, --macros MACROS
                        Directory containing TeX macros (paratext2.tex)
  -l, --logging         Output logging to terminal instead of logging window
  -e, --experimental    Enable experimental features
  --timeout TIMEOUT     xetex runtime timeout
  -T, --testing         Run in testing, output xdv
  -f FONTPATH, --fontpath FONTPATH
                        Directory of fonts to include (repeatable)
  --nofontcache         Don't use system fonts
  -S, --splash          Force splash screen
  -A ARCHIVE, --archive ARCHIVE
                        Create archive of project
  --nuseusfmerge        Use a different kind of merge for diglot
  --debug               Give debug output
```
It may not be as simple to create a suitable command line for a file. But I am open to adding code to parse a ptxprint.cfg full path and extract the project and config ids from it, to make the process a whole lot simpler. Of course if you can work out some Windows foo to not need that, that would be great. Also, let us know what you come up with and we'll see if we can add it to the installer. (Reg Key submissions welcomed).

**Comment by markpenny** [comment]
> When we first install PTXprint, I see a "readme" flash on the screen, which gives instructions on how to use it with a command line. But after it installs, I can't find that documentation anywhere.

Thanks for pointing that out. Yes, I need to include it in the installer so that it doesn't just display it, but copies it to the PTXprint folder as well. And I should probably put a link to it on the "...More Info" tab of the Viewer. (And the same goes for the release notes).

**Comment by markpenny** [comment]
Added these files to the installer for 0.9.1
![image](https://user-images.githubusercontent.com/12962708/89522233-fc3d5b80-d834-11ea-9d51-e2867625d84d.png)
(Not yet connected them to the UI though - maybe later)

**Comment by davidc86** [comment]
The documentation now goes into that folder nicely. Thanks. 
I'm still not seeing the link to this buried location in the "More info:" section in the Viewer tab under Folders & Links. All I'm seeing are links to folders for the  Current Project Path, Configuration Settings, and Output PDF Location. No link to the "...\PTXprint\ptxprint" documentation folder. As you said, it would be helpful to add it here. (The other URLs are helpful in there too, btw.)  

(I have not had time to experiment with getting the Config file to launch itself correctly yet.)

**Comment by markpenny** [comment]
I just tried associating .cfg files with "C:\Program Files\PTXprint\PTXprint.exe" and it load and seems to work fine if you double-click on the .cfg file in explorer. My only concern with this approach is that it creates a "palaso-usfm" folder in that location (when run), which we don't really want to be Send/Receiving when Paratext does its thing - so we may need to prevent that from happening.

Regarding the links, I've added a link which takes you to that folder and the tooltip tells you what to look for in there.
![image](https://user-images.githubusercontent.com/12962708/90085538-575adb00-dd6c-11ea-929d-ddbccb7c3300.png)

**Comment by markpenny** [comment]
@davidc86 We found and fixed another bug today relating to running .cfg files directly. This should be fixed in 1.2 when it comes out in a few days.

**Comment by davidc86** [comment]
Well, it kind of works in v1.1 -- it launches PTXprint with the correct config file loaded, but it won't print properly. It fails. But if I exit and reopen PTXprint directly, it REloads the last config file (thank you!) and properly formats the PDF -- me doing nothing at all other than exiting and reentering the application. So maybe that is the bug fixed in 1.2? Still, good progress. If I knew what the bug was you found in 1.1 maybe I can test in in 1.2 when I get it.

**Comment by mhosken** [comment]
I think this is falling foul of #223
